File: Arcade/Intro/javascript/digitDegree.js
function digitDegree(n) {
    let counter = 0;
    
    for(let sum = 0; (n+"").length > 1; counter++, sum = 0) {
        for(let e of (n+"")) sum += parseInt(e);
        n = sum;
    }
    
    return counter;
}

File: Arcade/Intro/Java/DigitDegree.java
final class DigitDegree {

	int digitDegree(int n) {
	    int counter = 0;
	    
	    for(; (n+"").length() > 1; counter++)
	        n = (n+"").chars().map(e -> new Integer((char)e+"")).sum();
	        
	    return counter;
	}
}